api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -269,13 +272,11 @@ public long getQueryBatchSize() {
                 .contactEmail(rs.getString("contact_email"))
                 .creationTime(offsetDateTimeUtc(rs.getTimestamp("creation_time")))
                 .currentPosition(rs.getString("current_position"))
-                .dataAccessLevel(dataAccessLevelFromStorage(rs.getShort("data_access_level")))
-                // TODO placeholder until we have a proper association of users to tiers
-                .accessTierShortNames(
-                    dataAccessLevelFromStorage(rs.getShort("data_access_level"))
-                            == DataAccessLevel.REGISTERED
-                        ? AccessTierService.REGISTERED_TIER_SHORT_NAME
-                        : "none")
+                // TODO remove.  until then, what we're interested in is registered vs not
+                .dataAccessLevel(
```

Comment:
Since we don't store Data Access Level on the user anymore, we fake it with this kluge.  (Soon we will not report it at all.  This allows for a transition.)